### ISSUE TITLE
Design backlog: change reporting attachments upload link to blue

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#765]
#### Describe the task

The link for the reporting attachments is black and don't meet accessibility standards. Change link colour to blue.

![Image](https://github.com/user-attachments/assets/2945442e-008c-4429-969a-8929d8b09faf)
